### DISCUSSION TITLE
请问能否增加一个GIF录屏功能？

### DISCUSSION THREAD

**Discussion by ContinueStudio** [shuax/MouseInc.Settings#135] [Ideas]
平常有学习各种三维软件，使用笔记记录不太方便。而每次打开GIF录屏软件再编辑再保存很麻烦，能否增加一个手势，直接添加选区就录屏，录完了直接保存本地。如果可以的话，那真的是太棒了。

**Comment by shuax** [comment]
这个比较困难，你用手势打开一个gif软件比较简单。
